co-simulation/lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/detector/DetectedObject.java

```diff
@@ -67,7 +67,7 @@ public final class DetectedObject implements Serializable {
      * @param angularVelocity   angular velocity of detected object in sensor/detector frame.
      * @param size              size of object including height,width and length.
      */
-    public DetectedObject(DetectionType type, double confidence, String sensorId, String projString, String objectId,
+    public DetectedObject(DetectionType type, double confidence, String sensorId, String projString, int objectId,
```

Comment:
The line 61 about the **objectId** description need to change. It is saying it is unique string ID